src/util/colorUtils.ts

```diff
@@ -1,9 +1,13 @@
 import { consoleWarn } from './console'
 
+export type RGB = number
+export type XYZ = [number, number, number]
+export type LAB = [number, number, number]
```

Comment:
Do non-distinct types help here? Typescript isn't nominally typed, so calling `number` something else doesn't change anything.

Reply:
It's only ever used from a namespace, so it's obvious what color format is returned

`LAB.fromXYZ(...`

Reply:
Although I guess vscode does show the actual type, so it isn't too bad. I'd definitely remove it from the public type file however.